content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,134 @@
+`iotbox-latest.zip`
+
+This image is compatible with all supported versions of Odoo.
+
+The images should be downloaded and extracted to a convenient file location.
+
+After this step is complete, insert the :abbr:`IoT (Internet of Things)` box's micro SD card into
+the computer or reader. Open *Etcher* and select :guilabel:`Flash from file`, then find and select
+the image just downloaded and extracted. Next, select the drive the image should be burned to.
+Lastly, click on :guilabel:`Flash` and wait for the process to finish.
+
+.. image:: updating_iot/etcher-app.png
+   :align: center
+   :alt: Balena's Etcher software dashboard.
+
+.. tip::
+   Balena's Etcher software also allows for the administrator to flash the :abbr:`SD (Secure
+   Digital)` card from a :abbr:`URL (Uniform Resource Locator)`. To flash from a :abbr:`URL (Uniform
+   Resource Locator)`, simply click :guilabel:`Flash from URL` instead of :guilabel:`Flash from
+   file`. Then enter the following: `http://nightly.odoo.com/master/iotbox/iotbox-latest.zip`.
+
+   .. image:: updating_iot/url-flash.png
+      :align: center
+      :alt:  A view of Balena's Etcher software, with the flash from URL option highlighted.
+
+.. note::
+   An alternative software for flashing the micro SD card is *Raspberry Pi Imager*. Download the
+   *Raspberry Pi* software `here <https://www.raspberrypi.com/software/>`_.
+
+Update from the IoT box home page
+=================================
+
+In the background, the :abbr:`IoT (Internet of Things)` box uses a version of Odoo code to run and
+connect to the Odoo database. This code may need to be updated in order for the :abbr:`IoT (Internet
+of Things)` box to operate effectively. This operation should be completed on a routine basis, to
+ensure the :abbr:`IoT (Internet of Things)` system and its processes stay up-to-date.
+
+Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to :menuselection:`IoT app -->
+IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT (Internet of Things)` box.
+Then click on :guilabel:`Update` (next to the version number).
+
+If a new version of the :abbr:`IoT (Internet of Things)` box image is available, an
+:guilabel:`Upgrade to _xx.xx_` button will appear at the bottom of the page. Click this button to
+upgrade the unit, at which point the :abbr:`IoT (Internet of Things)` box will flash itself to the
+newer version. All of the previous configurations will be saved.
+
+.. note::
+   This process can take more than 30 minutes. Do not turn off or unplug the :abbr:`IoT (Internet of
+   Things)` box as it would leave it in an inconsistent state. This means that the :abbr:`IoT
+   (Internet of Things)` box will need to be re-flashed with a new image.
+
+   .. seealso::
+      :ref:`flash_sdcard/etcher`.
+
+.. image:: updating_iot/flash-upgrade.png
+   :align: center
+   :alt: IoT box software upgrade in the IoT Box Home Page.
+
+.. _flash_sdcard/etcher:
+
+Handlers (drivers) update
```

Comment:
```suggestion
Handlers (drivers) update
```
Quick Question: I'm not very experienced with IoT boxes, and I'm curious if the term 'Handlers Code' should be possessive or not. In other words, should it be 'Handler's Code' or 'Handlers Code' -- whichever one it's *supposed* to be, be sure to implement the same version of it throughout the doc for consistency.

Reply:
Let's go with Handler's Code. I've updated this.